src/sentry/lang/native/processing.py

```diff
@@ -455,6 +480,49 @@ def process_native_stacktraces(symbolicator: Symbolicator, data: Any) -> Any:
     return data
 
 
+def emit_apple_symbol_stats(apple_symbol_stats, data):
+    os_name = get_path(data, "contexts", "os", "name") or get_path(
+        data, "contexts", "os", "raw_description"
+    )
+    os_version = get_path(data, "contexts", "os", "version")
+
+    if neither := apple_symbol_stats.get("neither"):
+        metrics.incr(
+            "apple_symbol_availability_v2",
+            amount=neither,
+            tags={"availability": "neither", "os_name": os_name, "os_version": os_version},
+            sample_rate=1.0,
+        )
+
+    # TODO: This seems to just be wrong
+    # We want mutual exclusion here, since we don't want to double count. E.g., an event has both symbols, so we
+    # count it both in `both` and `old` or `symx` which makes it impossible for us to know the percentage of events
+    # that matched both.
```

Comment:
I think this comment and the logic that was previously implemented is just wrong. The counters for `both`, `symx`, and `old` were already mutually exclusive, there was no double counting going on. Instead, the check meant that if _any_ debug image was found in both sources, _no_ images that were found in only one of them would be reported.